README.md

```diff
@@ -32,3 +33,159 @@ zopen build
 Note that running `zopen build` against your development repo will NOT install the code into your
 environment.
 
+
+# Developer Notes
+
+The detailed version of above.
+
+## Workflow
+
+The workflow for contributing to the meta repo.  This workflow assumes
+your installation of zopen is installed in `${HOME}`.
+
+### Step 1 - Update existing meta
+
+Update your z/OS Open Tools meta package before proceeding like so:
+
+```
+$ zopen upgrade meta -y
+```
+
+### Step 2 - Create the work dev repos for meta and metaport
+
+This sets up clones the two repos for development work. Note this workflow
+requires meta and metaport dirs to be named meta
+and metaport respectively.  
+
+Since merging to the main branch requires a pull request, this workflow shows
+how to contribute using a personal dev branch and then the use of github
+to perform a pull request.
+
+Its a common practice to work in the `$HOME/zopen/dev/` directory.
+
+```
+$ cd $HOME/zopen/dev
+$ git clone git@github.com:ZOSOpenTools/meta.git
+$ git clone git@github.com:ZOSOpenTools/metaport.git
+```
+
+### Step 3 - Create a branch in the dev version of meta for work
+
+For ease of use replace the text `YOURID` with your
+github userid.  This will create a branch which looks
+like: `xyzdev`.
+
+```
+$ cd $HOME/zopen/dev/meta
+$ git branch YOURUSERIDdev
+$ git checkout YOURUSERIDdev
+```
+
+Alternatively, you can create the branch and checkout with one command.
+
+```
+$ cd $HOME/zopen/dev/meta
+$ git checkout -b YOURUSERIDdev
+```
+
+
+### Step 4 - Create an environment script to setup for dev work
+
+This will allow you to enable and disable your work
+on meta.
+
+```
+$ cd $HOME/zopen/dev
+$ cat << EOF > setenv.sh
+> export ZOPEN_META_DEV_ROOT=$HOME/zopen/dev/meta
+> export ZOPEN_META_BRANCH=YOURUSERIDdev
+> EOF
+
+
+$ cat setenv.sh
+export ZOPEN_META_DEV_ROOT=${HOME}/zopen/dev/meta
+export ZOPEN_META_BRANCH=YOURUSERIDdev
+$
+```
+
+Source the script to have environment variables set in your current shell.
+
+```
+$ cd $HOME/zopen/dev
+$ . ./setenv.sh
+```
+
+Create the unset script to disalble using your sandbox.
+
+```
+$ cat << EOF > disable.sh
+> unset ZOPEN_META_DEV_ROOT
+> unset ZOPEN_META_BRANCH
+> EOF
+```
+
+It is used in similar fashion.
+
+
+
+### Step 5 - Build meta using branch YOURUSERIDdev
+
+```
+$ cd $HOME/zopen/dev/metaport
+$ zopen build
```

Comment:
I would add a little blurb here saying that zopen build builds your meta branch and also validates it via its test cases